docs/settings.rst

```diff
@@ -1179,8 +1179,7 @@ Seconds to store recursive packets in the :ref:`packet-cache`.
 ------------
 
 -  IP Address
-
-.. deprecated:: 4.1.0
+-  Removed in: 4.1.0
```

Comment:
We prefer to use the [Sphinx markup](http://www.sphinx-doc.org/en/stable/markup/para.html) for this. I suggest changing this to 

```rst
.. deprecated:: 4.1.0
  recursion is removed
```

With perhaps a link to the migration docs.

Reply:
There's a lot of other entries using "Removed in", why should this one as the only one use `.. deprecated::…` ?